account_bank_statement_import_txt_xlsx/models/account_bank_statement_import_sheet_mapping.py

```diff
@@ -107,10 +102,42 @@ class AccountBankStatementImportSheetMapping(models.Model):
             'transaction amount in original transaction currency from'
         ),
     )
+    amount_type = fields.Selection(
+        selection=[
+            ('simple_value', 'Simple value'),
+            ('absolute_value', 'Absolute value'),
+            ('distinct_credit_debit', 'Distinct Credit/debit Column'),
+        ],
+        string='Amount type',
+        required=True,
+        default="simple_value",
+        help=(
+            'simple_value: use signed amount in amount column\n'
+            'absolute_value: use same column for debit and credit\n'
+            '(absolute value + indicate sign)\n'
+            'distinct_credit_debit: use a distinct column for debit and credit'
+        ),
+    )
+    amount_column = fields.Char(
+        string='Amount column',
+        help=(
+            'Used if amount type is "Simple value" or "Absolute value"\n'
+            'Amount of transaction in journal\'s currency\n'
+            'Some statement formats use credit/debit columns'),
+    )
+    debit_column = fields.Char(
+        string='Debit column',
+        help='Used if amount type is "Distinct Credit/debit Column"\n',
+    )
+    credit_column = fields.Char(
+        string='Credit column',
+        help='Used if amount type is "Distinct Credit/debit Column"\n',
+    )
     debit_credit_column = fields.Char(
         string='Debit/credit column',
         help=(
-            'Some statement formats use absolute amount value and indicate sign'
+            'Used if amount type is "Absolute value"\n'
+            'Some statement formats use absolute amount value and indicate sign\n'
```

Comment:
I don't think `\n` are meaningful here, yet just a suggestion that this is not a list to be explicitly formatted